src/components/Dataset.js

```diff
@@ -313,228 +312,152 @@ export const Secretariatjr = [
       </>
     ),
   },
+];
+export const Secretariatjr = [
+  {
+    id: 1,
+    cName: "team-sec",
+    imgsrc: "https://res.cloudinary.com/dp92qug2f/image/upload/c_scale,w_360,f_auto,fl_lossy/v1676475732/events/nitsmun/aditi_MUN_o2zaxx.jpg",
+    alt: "Aditi Archita Khataniar",
+    name: "Aditi Archita Khataniar",
+    post: "Deputy Director General",
+    linkedin: "https://in.linkedin.com/in/aditi-archita-khataniar-2189a3226",
+    fb: "https://www.facebook.com/mumpyyyyyyy",
+    mail: "mailto:aditi.khataniar@gmail.com",
+  },
   {
     id: 2,
     cName: "team-sec",
-    imgsrc: "https://res.cloudinary.com/dp92qug2f/image/upload/c_scale,w_360,f_auto,fl_lossy/v1676476439/events/nitsmun/team/marufPadaya_faj39h.jpg",
-    alt: "Maruf Padaya",
-    name: "Maruf Padaya",
+    imgsrc: "https://res.cloudinary.com/dp92qug2f/image/upload/c_scale,w_360,f_auto,fl_lossy/v1676477071/events/nitsmun/team/Ronak_Jain_etgqal.jpg",
+    alt: "Ronak Jain",
+    name: "Ronak Jain",
     post: "Deputy Director General",
-    linkedin: "https://www.linkedin.com/in/maruf-padaya-a865151bb",
-    fb: "https://www.facebook.com/maruf.padaya",
-    mail: "mailto:marufpadaya786@gmail.com",
-    icon0: (
-      <>
-        {" "}
-        <button className="btn-icon3">
-          <FaFacebookF className="icon-color2" size={25} />
-        </button>{" "}
-      </>
-    ),
```

Comment:
these lines are responsible for fb link, so no need to delete. Update the fb link accordingly for every 3rd year except Deputy Director Technical Operations